src/refdb_fs.c

```diff
@@ -1430,18 +1442,38 @@ static int refdb_fs_backend__delete(
 	return error;
 }
 
+static int refdb_fs_backend__delete(
+	git_refdb_backend *_backend,
+	const char *ref_name,
+	const git_oid *old_id, const char *old_target)
+{
+	refdb_fs_backend *backend = (refdb_fs_backend *)_backend;
+	git_signature *who;
+	int error;
+
+	if ((error = git_reference__log_signature(&who, backend->repo)) < 0)
+		return error;
+
+	error = refdb_fs_backend__delete_impl(_backend, ref_name, who, NULL, old_id, old_target);
```

Comment:
Okay, so with this backwards-compatible change we'd be simply passing no message to the refdb deletion. Seems reasonable to me.